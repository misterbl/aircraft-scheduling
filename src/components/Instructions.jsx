import React from "react";

const Instructions = () => (
  <div className="instructions">
    <p>Click on an aircraft to see its available flights.</p>
    <p>Click on a flight in the flights list to add it to the rotation.</p>
    <p>Click on flight in the rotation to remove it</p>
  </div>
);

export default Instructions;
