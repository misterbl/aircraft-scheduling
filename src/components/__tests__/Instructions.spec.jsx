import * as React from "react";
import { shallow } from "enzyme";
import Instructions from "../Instructions";

describe("Instructions", () => {
  const wrapper = shallow(<Instructions />);
  it("matches the snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
