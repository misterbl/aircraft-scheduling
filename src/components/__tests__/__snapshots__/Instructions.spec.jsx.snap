// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Instructions matches the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Instructions />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <p>
          In the flights list, double click on a flight to add it to the rotation
        </p>,
        <p>
          In the rotation, double click on a flight to remove it 
        </p>,
      ],
      "className": "instructions",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "In the flights list, double click on a flight to add it to the rotation",
        },
        "ref": null,
        "rendered": "In the flights list, double click on a flight to add it to the rotation",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "In the rotation, double click on a flight to remove it ",
        },
        "ref": null,
        "rendered": "In the rotation, double click on a flight to remove it ",
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <p>
            In the flights list, double click on a flight to add it to the rotation
          </p>,
          <p>
            In the rotation, double click on a flight to remove it 
          </p>,
        ],
        "className": "instructions",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "In the flights list, double click on a flight to add it to the rotation",
          },
          "ref": null,
          "rendered": "In the flights list, double click on a flight to add it to the rotation",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "In the rotation, double click on a flight to remove it ",
          },
          "ref": null,
          "rendered": "In the rotation, double click on a flight to remove it ",
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
